zookeeper-api/src/main/java/org/apache/helix/zookeeper/impl/factory/SharedZkClientFactory.java

```diff
@@ -44,15 +44,17 @@ protected SharedZkClientFactory() {
   @Override
   public RealmAwareZkClient buildZkClient(
       RealmAwareZkClient.RealmAwareZkConnectionConfig connectionConfig,
-      RealmAwareZkClient.RealmAwareZkClientConfig clientConfig) {
+      RealmAwareZkClient.RealmAwareZkClientConfig clientConfig,
```

Comment:
This is PR for dedicated zkclient factory, right?

Reply:
There was a change in the interface. SharedZkClient implements that interface. So this change is just a signature change, not logical change. 

This change is necessary.